BuildingSync.xsd

```diff
@@ -15254,6 +15254,20 @@
                   <xs:documentation>The story of the given floor area type.</xs:documentation>
                 </xs:annotation>
               </xs:element>
+              <xs:element name="ExcludedSectionIDs" minOccurs="0">
+                <xs:annotation>
+                  <xs:documentation>Links to Sections not included in the floor area calculation.</xs:documentation>
```

Comment:
`Links to Sections with floor areas which are not included in this floor area calculation.`